plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -318,8 +320,17 @@ void GSSettingsDlg::UpdateRenderers()
 	}
 	else
 	{
-		GSRendererType ini_renderer = GSRendererType(theApp.GetConfigI("Renderer"));
-		renderer_setting = (ini_renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : ini_renderer;
+		renderer_setting = GSRendererType(theApp.GetConfigI("Renderer"));
+
+		if (renderer_setting == GSRendererType::Undefined)
+		{
+			renderer_setting = GSUtil::GetBestRenderer();
+
+			if (renderer_setting == GSRendererType::OGL_HW)
+			{
```

Comment:
I would prefer if the redundant braces are removed at the ``if (renderer_setting == GSRendererType::OGL_HW)`` check. too much braces are not always nice :)